app/models/user.rb

```diff
@@ -417,6 +417,7 @@ def non_banished_username
   def subscribe_to_mailchimp_newsletter
     return unless email.present? && email.include?("@")
     return if saved_changes["unconfirmed_email"] && saved_changes["confirmation_sent_at"]
+    return if saved_changes.key?(:user_email) || saved_changes.key?(:email_newsletter)
```

Comment:
Correct me if I'm wrong, so this would exit early if saved_changes include `:email_newsletter`. That would be a problem if a user decides to unsubscribe on our platform as our platform won't update MailChimp.

Reply:
That should be an `unless` 🤦‍♂